assets/scss/00-base/_fonts.scss

```diff
@@ -132,3 +132,171 @@
   font-style: italic;
   font-display: fallback;
 }
+
+// source-sans-pro-200 - latin.
+@font-face {
+  font-family: 'Source Sans Pro';
+  font-style: normal;
+  font-weight: 200;
+  src: url('../fonts/source-sans-pro-200.eot'); // IE9 Compat Modes.

```

Comment:
can you set the relative path with the assets-path variable? that's how patternlab and react are pointing to the assets, see example --> https://github.com/massgov/mayflower/blob/1a480c78eb245da4d70c7b01f2d7c395c76198e3/assets/scss/00-base/_fonts.scss#L11